rc/tools/go/gopls.kak

```diff
@@ -0,0 +1,49 @@
+hook -once global BufSetOption filetype=go %{
+    require-module gopls
+}
+
+evaluate-commands %sh{
+    if ! command -v gopls > /dev/null 2>&1; then
+        echo "fail Please install gopls or add to PATH!"
```

Comment:
I wonder if it's better to print that to the debug buffer, like go-tools does.

Reply:
I like the big fat red error message as without gopls is useless. On the other hand if we autoload it with every go file it can be annoying if someone do not want to use it.
So I think the main question is if we want to autoload gopls.kak for go files, or we just keep the gopls-mode command. I prefer the latter, as gopls has no interactive part, so users will invoke it manually every time, autoloading is not useful IMO.